spec/grape/kaminari_spec.rb

```diff
@@ -8,7 +8,7 @@
   end
 
   def declared_params
-    subject.namespace_stackable(:declared_params).flatten
+    Grape::Validations::ParamsScope::Attr.attrs_keys(subject.namespace_stackable(:declared_params).flatten)
```

Comment:
using method from https://github.com/ruby-grape/grape/blob/master/lib/grape/validations/params_scope.rb#L25
as subject.namespace_stackable(:declared_params) returning array of Grape::Validations::ParamsScope::Attr instances.